src/__test__/components/plots/ContinuousEmbeddingPlot.test.jsx

```diff
@@ -91,6 +93,7 @@ describe('Continuous embedding plot', () => {
           experimentId={experimentId}
           config={config}
           plotUuid={plotUuid}
+          truncatedPlotData={mockedStore.genes.expression.data[shownGene].expression}
```

Comment:
Maybe you can put `mockedStore.genes.expression` into a variable, so these props across the file are shorter